src/Provisioning/ProvisionAssets.ps1

```diff
@@ -76,7 +77,18 @@ Param
 $ErrorActionPreference = "Stop"
 
 # Check the azure module is installed
-Import-Module azure
+if(-not(Get-Module -name "Azure")) 
+{ 
+    if(Get-Module -ListAvailable | Where-Object { $_.name -eq "Azure" }) 
+    { 
+        Import-Module Azure
+    }
+    else
+    {
+        "Windows Azure Powershell has not been installed."
```

Comment:
Oh, I guess it executes as early as it can. Am I correct in that the inputs have to be collected first?